actions/hadoop.conf

```diff
@@ -4,10 +4,6 @@ hibench.hadoop.executable     ${hibench.hadoop.home}/bin/hadoop
 # Hadoop configraution directory
 hibench.hadoop.configure.dir  ${hibench.hadoop.home}/etc/hadoop
 
-# The root HDFS path to store HiBench data
-hibench.hdfs.master       hdfs://localhost:9000
-
```

Comment:
Why removing this? Can you just update the value?

Reply:
Removing this  because of we need using hostname instead of localhost, then we will set hibench.hdfs.master with setup-hibench.sh.